inc/task.class.php

```diff
@@ -221,4 +234,118 @@ static function getTaskActionButton($url, $task_id, $status) {
+      echo "<tr class='headerRow'>";
+      echo '<th colspan="4">' . __('New target', 'openvas') . '</th>';
+      echo '</tr>';
+
+      if (!PluginOpenvasOmp::ping()) {
+         echo "<tr class='tab_bg_1' align='center'>";
+         echo "<th>" . __('Cannot contact OpenVAS', 'openvas') . "</th>";
+         echo "</tr>";
+      } else {
+
+         echo '<tr class="tab_bg_1">';
+         echo '<td colspan="4">';
+         Dropdown::showSelectItemFromItemtypes(array('itemtype_name'       => 'item',
+                                                     'items_id_name'       => 'add_items_id',
+                                                     'itemtypes'           => $CFG_GLPI["asset_types"]));
+         echo '</td>';
+         echo '</tr>';
+
+         echo '<tr class="tab_bg_1">';
+         echo "<td>";
+         echo __('Credentials for ', 'openvas') . __('SSH');
+         echo "</td>";
+         echo "<td>";
+         PluginOpenvasOmp::displayDropdown(PluginOpenvasOmp::CREDENTIAL, 'credential', 'credential_ssh');
+         echo "</td>";
+         echo "<td>";
+         echo __('On port:', 'openvas');
+         echo "</td>";
+         echo "<td>";
+         echo "<input id='text' type='text' name='port' value='' class='autocompletion-text-field'>";
+         echo "</td>";
+         echo '</tr>';
+
+         echo '<tr class="tab_bg_1">';
+         echo "<td>";
+         echo __('Credentials for ', 'openvas') . __('SMB');
+         echo "</td>";
+         echo "<td colspan='3'>";
+         PluginOpenvasOmp::displayDropdown(PluginOpenvasOmp::CREDENTIAL, 'credential', 'credential_smb');
+         echo "</td>";
+         echo '</tr>';
+
+         echo "<tr class='tab_bg_2 center'><td colspan='4'>";
+         echo Html::submit(_x('button', 'Add'), array('name' => 'add'));
+         echo "</td></tr>";
+         echo "</table>";
+         Html::closeForm();
+
+      }
+
+   }
+
+
+   function formForSchedule() {
+
+      global $CFG_GLPI;
+
+      echo "<form name='addschedule' method='post' action='" . $CFG_GLPI['root_doc'] . "/plugins/openvas/front/schedule.form.php'>";
+
+      echo "<table class='tab_cadre_fixe'>";
+      echo "<tr class='headerRow'>";
+      echo '<th colspan="2">' . __('New schedule', 'openvas') . '</th>';
+      echo '</tr>';
+
+      if (!PluginOpenvasOmp::ping()) {
+         echo "<tr class='tab_bg_1' align='center'>";
+         echo "<th>" . __('Cannot contact OpenVAS', 'openvas') . "</th>";
+         echo "</tr>";
+      } else {
+
+         echo '<tr class="tab_bg_1">';
+         echo "<td>";
+         echo __('Name');
+         echo "</td>";
+         echo "<td>";
+         echo "<input type='text' name='name' value=''>";
+         echo "</td>";
+         echo '</tr>';
+
+         echo '<tr class="tab_bg_1">';
+         echo "<td>";
+         echo __('First time', 'openvas');
+         echo "</td>";
+         echo "<td>";
+         Html::showDateFormItem("date", "", false);
```

Comment:
nat available anymore in glpi 9.2, please use ``Html::showDateField(...)`` instead